index.js

```diff
@@ -0,0 +1,203 @@
+document.addEventListener("DOMContentLoaded", () => {
+  recipeObject.forEach((recipe) => {
+    setRecipe(recipe);
+  });
+});
+
+
+
+function setRecipe(recipe) {
+  setRecipeTitle(recipe);
+  setRecipeImage(recipe);
+  setRecipeDescription(recipe);
+  setIngredients(recipe);
+  setPreparationSteps(recipe);
+}
+
+const recipeObject = [
+  {
+    id: 1,
+    title: "Gløgg",
+    picture_url:
+      "https://upload.wikimedia.org/wikipedia/commons/thumb/e/e2/Gl%C3%B6gg_kastrull.JPG/800px-Gl%C3%B6gg_kastrull.JPG",
+    description:
+      "Flæskesteg, a Danish Crisp Pork, is a cherished dish deeply rooted in Danish culinary tradition. It's not just a meal; it's a centerpiece of celebration, often enjoyed during holidays like Christmas. This dish boasts a succulent pork roast with a perfectly crispy and crackling skin, making it a delight for both the palate and the senses.",
+    ingredients: [
+      { NAME: "Orange zest", AMOUNT: "0.5" },
+      { NAME: "Water", AMOUNT: "200 ml" },
+      { NAME: "Sugar", AMOUNT: "275 g" },
+      { NAME: "Whole cloves", AMOUNT: "5" },
+      { NAME: "Cinnamon sticks", AMOUNT: "2" },
+      { NAME: "Spice", AMOUNT: undefined },
+      { NAME: "Bottle of red wine", AMOUNT: "1" },
+      { NAME: "Raisins", AMOUNT: "100 g" },
+      { NAME: "Slipped Almonds", AMOUNT: "50 g" },
+    ],
+    preparationSteps: ["Mix everything, heat it, and you are good to go!"],
+  },
+];
+
+function setRecipeTitle(recipe) {
+  const recipeTitle = document.getElementById("recipe-title");
+  if (recipeTitle) {
+    recipeTitle.textContent = recipe.title;
+  } else {
+    console.error("Element #recipe-title not found");
+  }
+}
+
+function setRecipeImage(recipe) {
+  const recipeImage = document.getElementById("recipe-image");
+  if (recipeImage) {
+    recipeImage.src = recipe.picture_url;
+    recipeImage.alt = recipe.title;
+  } else {
+    console.error("Element #recipe-image not found");
+  }
+}
+
+function setRecipeDescription(recipe) {
+  const recipeDescription = document.getElementById("recipe-description");
+  if (recipeDescription) {
+    recipeDescription.innerText = recipe.description;
+  } else {
+    console.error("Element #recipe-description not found");
+  }
+}
+
+function setIngredients(recipe) {
+  const ingredientList = document.querySelector("#recipe-ingredients ul");
+  if (ingredientList) {
+    ingredientList.innerHTML = "";
+    recipe.ingredients.forEach((ingredient) => {
+      const li = document.createElement("li");
+      if (ingredient.NAME && ingredient.AMOUNT) {
+        li.textContent = `${ingredient.NAME}: ${ingredient.AMOUNT}`;
+      } else if (ingredient.NAME && !ingredient.AMOUNT) {
+        li.textContent = `${ingredient.NAME}: appropriate`;
+      }
```

Comment:
what is the purpose of this else statement? :)

Reply:
If user didnot provide/know the exact amount of the ingredient (just like salt/peper which you mentioned in the next comment), just specify an amount as "appropriate." Perhaps I used the wrong word? which caused confusion in English. 🤧

Reply:
Yeah, that makes sense. 👍